gammapy/maps/geom.py

```diff
@@ -935,8 +935,34 @@ def from_hdulist(cls, hdulist, hdu=None, hdu_bands=None):
 
         return cls.from_header(hdu.header, hdu_bands)
 
+    def make_bands_hdu(self, hdu=None, hdu_skymap=None, conv=None):
+        conv = self.conv if conv is None else conv
+        header = fits.Header()
+        self._fill_header_from_axes(header)
+        axis_names = None
+
+        # FIXME: Check whether convention is compatible with
+        # dimensionality of geometry
+
+        if conv == 'fgst-ccube':
+            hdu = 'EBOUNDS'
+            axis_names = ['energy']
+        elif conv == 'fgst-template':
+            hdu = 'ENERGIES'
+            axis_names = ['energy']
+        elif hdu is None and conv == 'gadf':
+            if hdu_skymap:
+                hdu = '{}_{}'.format(hdu_skymap, 'BANDS')
+            else:
+                hdu = 'BANDS'
```

Comment:
Probably should end with an else ValueErrors here, to give good error message if the user mistypes the `conv` string?

Reply:
Agreed.  Although at some point we should probably implement some common registry for allowed `conv` values.